docs/source/upcoming_release_notes/1001-jj-slits.rst

```diff
@@ -0,0 +1,30 @@
+1001 jj-slits
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- New typhos screen for newly created JJSlits class
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- JJSlits class within slits.py
```

Comment:
Probably needs more text here, just like in the empty class docstring: what exactly is this for? The new beckhoff controlled slits?